interface/forms/LBF/new.php

```diff
@@ -1598,6 +1756,9 @@ function warehouse_changed(sel) {
             <?php if (!$from_trend_form) { // end row and container divs ?>
         </div>
     </div>
+
+    <p style='text-align:center' class='small'>Rev. <?php echo substr($grp_last_update, 0, 10); ?></p>
```

Comment:
not sure where this variable comes from, but would be safe and surround it with a text()

Reply:
Yeah it comes from layout_group_properties.grp_last_update. Thanks.